prow/config.yaml

```diff
@@ -171,7 +171,12 @@ presubmits:
     trigger: "(?m)^/(retest|test (all|istio-presubmit))?,?(\\s+|$)"
     branches:
     - master
-    - release-0.5.1
+    - release-0.5
+    - release-0.6
```

Comment:
I was not sure whether this would resort to no op if branch does not exist. Maybe add those branches in after we make them?

Reply:
it should no-op, I looked at the code. Trying to add more so that we will need less manual intervention when creating patch release.